src/pages/components/Card/Card.tsx

```diff
@@ -17,18 +18,16 @@ const Card = ({
       className={`w-full flex flex-col items-center py-4 cursor-default
       shadow-md rounded-lg px-5 my-10 bg-gradient-to-t from-gray-50 to-white`}
     >
-      <div className="flex flex-row justif-start items-center space-x-2">
-        <span className="rounded-full text-3xl w-16 bg-white my-4 justify-center text-center">
-          {icon}
-        </span>
+      <div className="flex flex-row items-center space-x-2 justif-start">
+        <span className="justify-center w-16 my-4 text-3xl text-center bg-white rounded-full">{icon}</span>
         <div>
           <h1 className="text-2xl font-bold text-gray-900">{title}</h1>
           <p
             style={{
               maxHeight: 200,
               maxWidth: 400,
             }}
-            className="text-l text-gray-500"
+            className="text-gray-500 text-l"
```

Comment:
ideally we wouldn't want to change anything within these files except if it relates to types

Reply:
This is automatically done by prettier